content/chapter-03/ICanNotConnectLab.md

```diff
@@ -0,0 +1,44 @@
+# Lab: What to do when I cannot connect?
+
+* In this lab, we are going to deploy another application: `count_lines`
+* This application will consume the book content provided by `get_a_book` and return the line count.
+
+
+* Start `get_a_book` application
+  * Open an ssh session into the application instance
+  * Change directory into the app folder to run it
+    * `cd get_a_book`
+    * `npm start`
+  
+  
+* `count_lines` application instance
+  * Create another small install to install `count_lines` application and monitor it
+  * ssh into the new instance and install `node` and `npm`:
+    * `sudo apt-get update`
+    * `sudo apt-get install npm`
+    * `curl -sL https://deb.nodesource.com/setup_9.x | sudo -E bash -`
+    * `sudo apt-get install -y nodejs`
+  * Install `unzip` utility: `sudo apt install unzip`
+  * Download the application:
+    * `curl -L -O https://raw.githubusercontent.com/vkhazin/cloudmigration-courseware/master/lab_support/count_lines.zip`
+  * Unzip the app: `unzip count_lines.zip`
+  * This application will open the port 10000. Ensure you configure your network security correctly to make that port accessible
+  * Change directory into the app folder to run it
+    * `cd count_lines`
+    * `npm start <get_a_book instance public ip or dns>:9000`
```

Comment:
Hint's where to find this information, portal console or ifconfig on the instance

Reply:
Done